test/unit/object/test_comment.py

```diff
@@ -0,0 +1,64 @@
+# coding: utf-8
+
+from __future__ import unicode_literals
+import json
+
+from boxsdk.object.comment import Comment
+
+
+# pylint:disable=protected-access
+# pylint:disable=redefined-outer-name
+
+def test_reply(test_comment, mock_box_session, comment_params):
+    expected_url = mock_box_session.get_url('comments')
+    (message_type, message) = comment_params
+    expected_data = {
+        message_type: message,
+        'item': {
+            'type': 'comment',
+            'id': test_comment.object_id
+        }
+    }
+    mock_box_session.post.return_value.json.return_value = {
+        'type': 'comment',
+        'id': '12345',
+        message_type: message
+    }
+    reply_comment = test_comment.reply(message)
+    mock_box_session.post.assert_called_once_with(expected_url, data=json.dumps(expected_data))
+    assert reply_comment.object_id == '12345'
+
+
+def test_edit(test_comment, mock_box_session, comment_params):
+    expected_url = mock_box_session.get_url('comments', test_comment.object_id)
+    (message_type, message) = comment_params
+    expected_data = {
+        message_type: message,
+    }
+    mock_box_session.post.return_value.json.return_value = {
```

Comment:
Shouldn't this be `put`, not `post`?

Apparently it doesn't matter since it didn't cause a test failure. But that's probably because there aren't any assertions on the response type/data.